glassfish.tooling/test/unit/src/org/netbeans/modules/glassfish/tooling/GlassFish.properties-dist

```diff
@@ -1,3 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
```

Comment:
Please remove the first and the end lines because the other properties files don't have them. e.g. see
https://github.com/apache/incubator-netbeans/blob/6a5e347fac2acf11211e9ef67e491cd84b0f4850/glassfish.tooling/nbproject/project.properties#L1-L18

The others properties files as well.